robotbase/src/main/java/com/team2357/frc2022/shuffleboard/CargoNumberWidget.java

```diff
@@ -0,0 +1,33 @@
+package com.team2357.frc2022.shuffleboard;
```

Comment:
Merge this and CargoCountWidget and update to use the functional interfaces for the arduino sensors created in RobotContainer.java